localbuild.xml

```diff
@@ -434,7 +434,7 @@
   <project path="external/python/parse_type" name="platform/external/python/parse_type" groups="vts,pdk" />
   <project path="external/python/pyasn1" name="platform/external/python/pyasn1" groups="vts,pdk" />
   <project path="external/python/pyasn1-modules" name="platform/external/python/pyasn1-modules" groups="vts,pdk" />
-  <project path="external/python/pybind11" name="platform/external/python/pybind11" groups="pdk" />
+  <project path="external/python/pybind11" name="platform/external/python/pybind11" groups="pdk" revision="android12-release"/>
```

Comment:
The tip of `android12-release` is not 8d0f6cb976bd7e50c3ce4ee9d9683c702cd7db32, but this hash is in tagged manifest.

Reply:
Suggestion to freeze revision at "8d0f6cb976bd7e50c3ce4ee9d9683c702cd7db32", because `android12-release` is a branch, and if we set such principle for the future, it will increase points of failure when a branch moves on to introduce breakage.

Reply:
Changed to freeze at "b800b1b1efc6a89441803ebba769ec7fc004e356"